src/Microsoft.DotNet.ProjectJsonMigration/MSBuildExtensions.cs

```diff
@@ -27,14 +27,18 @@ public static bool IsEquivalentTo(this ProjectItemElement item, ProjectItemEleme
             // Different includes
             if (item.IntersectIncludes(otherItem).Count() != item.Includes().Count())
             {
+#if !DISABLE_TRACE
```

Comment:
This feels wrong to me. Feels like you have a different underlying problem that is not being addressed here. Why do we need an #if in the product code to make a test compile?